src/integration/common/header.js

```diff
@@ -15,15 +15,13 @@ export default () => {
       expect(langCode).toBeInTheDocument();
     });
 
-    it('I can see the navigation', () => {
-      const navigationItemATag = document.querySelector(
-        'header nav ul > li > a',
-      );
-
-      expect(navigationItemATag).toBeInTheDocument();
-      expect(navigationItemATag.getAttribute('href')).toBeTruthy();
-      expect(navigationItemATag.textContent).toBeTruthy();
-      expect(navigationItemATag.textContent).toMatchSnapshot();
+    const navigationLinks = document.querySelectorAll('header nav ul > li > a');
+    navigationLinks.forEach(navigationLink => {
+      it(`I can see a navigation link: ${navigationLink.textContent}`, () => {
```

Comment:
this test would pass if the link text changed unexpectedly. Could I suggest something like:

```js
describe('Navigation links', () => {
  const navLinks = document.querySelectorAll('header nav ul > li > a');

  navLinks.forEach(navLink => {
    const navLinkText = navLink.textContent;
    const navLinkUrl = navLink.getAttribute('href');

    it('should be in the document', () => {
      expect(navLink).toBeInTheDocument();
    });

    it('should contain text', () => {
      expect(navLinkText).toBeTruthy();
    });

    it('should match text', () => {
      expect(navLinkText).toMatchSnapshot();
    });

    it('should match link url', () => {
      expect(navLinkUrl).toEqual('content');
    });
  });
});
```